src/onboarding/index.ts

```diff
@@ -3,8 +3,12 @@ import { ExperimentConfigs } from 'src/statsig/constants'
 import { StatsigExperiments } from 'src/statsig/types'
 
 export function getOnboardingExperimentParams() {
+  const { onboardingNameScreenEnabled, chooseAdventureEnabled } = getExperimentParams(
```

Comment:
The way that we're accessing experiment parameters here (and elsewhere) is kind of confusing to me.

`getOnboardingExperimentParams` now returns parameters for multiple experiments. For `RECOVERY_PHRASE_IN_ONBOARDING`,  it returns _all_ parameters; anywhere we want to access a parameter for this experiment (whether or not it's used in onboarding, per se) we do so through this function. For `CHOOSE_YOUR_ADVENTURE`, we only return a subset of the available experiment params, seemingly, just the ones that are actually used in onboarding. If we want one of the other parameters, we have to call `getExperimentParams` directly. 

I'm wondering if we should consider one of these alternatives instead:
* expose _all_ parameters for both experiments in `getOnboardingExperimentParams`, and the caller just picks what it needs.
* Have two different functions, e.g., `getOnboardingRecoveryExperimentParams` and `getChooseYourAdventureExperimentParams`, and have both of them return all their respective parameters.

It mainly seems weird to me that we can get _most_ of the experiment params through this function, but _some_ have to gotten directly from a call to `getExperimentParams`. I think I like the first option above more, but curious to hear other thoughts/if I'm missing something obvious.

Reply:
I see what you mean. I think what you suggested as the first option is probably cleaner.
> expose all parameters for both experiments in getOnboardingExperimentParams, and the caller just picks what it needs.

I'll update for that

Reply:
+1 to Charlie's comment. Why not just directly call getExperiments like you did in WalletHome and avoid this extra layer?

Reply:
Didn't consider the exposure event issue -- this solution makes sense to me!